protocol/go.mod

```diff
@@ -427,11 +427,11 @@ replace (
 	// should use v0.11.0. The Cosmos build fails with types/context.go:65:29: undefined: comet.BlockInfo otherwise.
 	cosmossdk.io/core => cosmossdk.io/core v0.11.0
 	// Use dYdX fork of Cosmos SDK/store
-	cosmossdk.io/store => github.com/dydxprotocol/cosmos-sdk/store v1.0.3-0.20240326192503-dd116391188d
+	cosmossdk.io/store => github.com/dydxprotocol/cosmos-sdk/store v1.0.3-0.20240515183437-194a18ccaefa
```

Comment:
No, I haven't merged that PR yet. I would change this PR to point to a commit on dydx-fork-v0.50.5 before merging. This was just temporary since I was working on these PRs in parallel.